tests/static-analysis/empty.php

```diff
@@ -14,15 +14,15 @@
 /**
  * @param Collection<int, int> $collection
  */
-function checkNumeric(Collection $collection): void
+function empty_checkNumeric(Collection $collection): void
```

Comment:
I realised these functions need unique names because they're not under a namespace. I only got away with it so far because they had the same type annotations so the analysers didn't complain before 😄